ui/src/pages/transformations/formula.js

```diff
@@ -69,6 +73,10 @@ export const formula = (app) => {
     } = config;
 
     columns = availableColumns;
+    suggestor = { '.': availableColumns.map(col => ({
+      title: col.name,
+      value: `.${col.name}`,
```

Comment:
The value should be `."${col.name}"`

Because if the column name has spaces within it the way to allow jq parser to understand the input field if wrapping it within double quotes.